acceptancetests/assess_persistent_storage.py

```diff
@@ -165,13 +281,14 @@ def assess_persistent_storage(client):
     # Based on the test spec, persistent storage need to be tested on both
     # LXD and AWS and ebs volume is unavailable on LXD, a switcher is
     # required here to decide which test should be run.
-    environment = os.getenv('ENV', default='parallel-lxd')
```

Comment:
This is a temporary function which I will modify or just remove it. The current status is persistent storage doesn't support lxd - only rootfs is there and it's not persistent; but support all others. The correct way to handle it is to run on all others except lxd.